File: benchmarks/scenarios/payments-boundary/anma/src/domains/billing/service.py
"""billing public surface. May import accounts."""
from domains.accounts.service import get_user

def create_invoice(user_id: str, amount: int) -> dict:
    user = get_user(user_id)
    return {"user": user["id"], "amount": amount}

def total_invoiced(user_id: str) -> int:
    return 0

File: benchmarks/scenarios/ts-payments/control/src/domains/billing/service.ts
// billing public surface. May import accounts (allowed).
import { getUser } from "../accounts/service";

export function createInvoice(userId: string, amount: number) {
  const user = getUser(userId);
  return { user: user.id, amount };
}

export function totalInvoiced(_userId: string): number {
  return 0;
}
